cypress/integration/data-browser/graphs.spec.js

```diff
@@ -1,171 +1,154 @@
-import {
-  firstGraphURLQuery,
-  firstGraphURL,
-  ALL_GRAPHS_ENDPOINT,
-  buildURL,
-  firstGraphEndpoint,
-  devURL,
-} from "../../fixtures/mockGraphData-";
-
-import { isCI } from "../../support/helpers";
-
-const { HOST, ENVIRONMENT } = Cypress.env();
-
-let baseURLToVisit = isCI(ENVIRONMENT) ? "http://localhost:3000" : HOST;
-
-describe.skip("General Tests", () => {
-  it("URL Re-direct which contains base period selections and series", () => {
-    cy.visit(`${baseURLToVisit}/data-browser/graphs/quarterly`);
-    cy.get(".react-select__graph__control")
-      .click(0, 0, { force: true })
-      .find('[class*="-singleValue"]')
-      .eq(0)
-      .contains("How has the number of applications changed?")
-      .click(0, 0, { force: true });
-    cy.url().should("include", firstGraphURLQuery);
-    cy.url().should("eq", baseURLToVisit + firstGraphURL);
-  });
-
-  it("Checks <GraphsHeader/> component if overview props was not sent to the component", () => {
-    cy.visit(`${baseURLToVisit}/data-browser/graphs/quarterly`);
-    let overviewHeader = cy.get(".Graphs > :nth-child(2)");
-    if (!ALL_GRAPHS_ENDPOINT.endpoint) {
-      overviewHeader.contains(
-        "The following graphs present data for the financial institutions reporting HMDA quarterly data."
-      );
-    }
-  });
-
-  it("Checks <GraphsHeader/> component if overview props was sent to the component", () => {
-    let overviewHeader = cy.get(".Graphs > :nth-child(2)");
-    if (ALL_GRAPHS_ENDPOINT.endpoint) {
-      overviewHeader.contains(
-        ALL_GRAPHS_ENDPOINT.overview +
-          "Though the graphs provide some insight into trends for these institutions, they should not be taken to represent the behavior of all mortgage lenders during the relevant period." +
-          "Use the menu below to select a graph."
-      );
-    }
-  });
-
-  it("Selects first graph from graph drop-down list and checks url contains query parameters", () => {
-    cy.get(".react-select__graph__control")
-      .click(0, 0, { force: true })
-      .find('[class*="-singleValue"]')
-      .eq(0)
-      .contains("How has the number of applications changed?")
-      .click(0, 0, { force: true });
-    cy.url().should("include", firstGraphURLQuery);
-    cy.url().should("eq", baseURLToVisit + firstGraphURL);
-  });
+import { isCI } from "../../support/helpers"
+
+const { HOST, ENVIRONMENT } = Cypress.env()
+
+let baseURLToVisit = isCI(ENVIRONMENT) ? "http://localhost:3000" : HOST
+
+describe("General Tests", () => {
+  it.skip("Checks <GraphsHeader/> component if overview props was not sent to the component", () => {
```

Comment:
Please make sure to remove any `.skip` for tests we DO want to run.

Reply:
Removed all the `.skip`.